tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/Containers/AerospikeFixture.cs

```diff
@@ -28,7 +30,8 @@ protected override async Task InitializeResources(Action<string, object> registe
         // (6.3.0.5 at time of issue) causes 'Server memory error' and flake
         var container = new ContainerBuilder()
                        .WithImage("aerospike/aerospike-server:6.2.0.6")
-                       .WithPortBinding(3000, true)
+                       .WithPortBinding(AerospikePort, true)
+                       .WithWaitStrategy(Wait.ForUnixContainer().UntilPortIsAvailable(AerospikePort))
```

Comment:
Should it actually be this? 🤔 
```suggestion
                       .WithWaitStrategy(Wait.ForUnixContainer().UntilPortIsAvailable(Container.GetMappedPublicPort(AerospikePort)))
```